onnx/version_converter/adapters/argmax_argmin_12_11.h

```diff
@@ -0,0 +1,26 @@
+// Adapter for ArgMax/ArgMin in default domain from version 12 to 11
+
+#pragma once
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+class ArgMaxArgMin_12_11 final : public Adapter {
+  public:
+    explicit ArgMaxArgMin_12_11(const std::string& op_name)
+      : Adapter(op_name, OpSetID(12), OpSetID(11)) {}
+
+    void adapt_argmax_argmin_12_11(std::shared_ptr<Graph> graph, Node* node) const {
+    	Symbol select_last_index = Symbol("select_last_index");
+	    if (node->hasAttribute(select_last_index)) {
+	      ONNX_ASSERTM(node->i(select_last_index) == 0,
+	          "select_last_index != 0 is not supported in version 11");
```

Comment:
nit: Change error message to something like : Opset version 11 only supports select_last_index == 0?

Reply:
Done